src/main/java/com/imsweb/geocoder/entity/Session.java

```diff
@@ -276,6 +293,7 @@ public void deserializeFromMap(Map<String, Object> map) {
         setNumNoResultLines((Integer)map.get(_KEY_NUM_NO_RESULT_LINES));
         setCurrentLineNumber((Integer)map.get(_KEY_CURRENT_LINE_NUMBER));
         setSkippedMode((Boolean)map.get(_KEY_SKIPPED_MODE));
+        setIncludedCensusYears((Set<String>)map.get(_KEY_INCLUDED_CENSUS_YEARS));
```

Comment:
Are you properly handling a null value for the set? It could happen if someone deserialize and old result with this new code...